src/game/AI/ScriptDevAI/scripts/northrend/ulduar/ulduar/boss_ignis.cpp

```diff
@@ -120,13 +120,15 @@ struct boss_ignisAI : public BossAI
     void EnterEvadeMode() override
     {
         BossAI::EnterEvadeMode();
+        m_creature->GetMap()->GetVariableManager().SetVariable(WORLD_STATE_CUSTOM_RESPAWN_IGNIS_ADDS, 1);
         auto* ignisAdds = m_creature->GetMap()->GetCreatures(ULDUAR_IGNIS_ADDS);
+        if ((*ignisAdds).empty())
```

Comment:
If i remember my own code correctly, ignisAdds will be nullptr. This is kaboom code :D